helpme/helpme.py

```diff
@@ -976,6 +979,13 @@ def run(self):
 
 @plugin.init()
 def init(options, configuration, plugin):
+    rpchelp = plugin.rpc.help().get('help')
+    # detect if server cli has moved `listpeers.channels[]` to `listpeerchannels`
+    # See https://github.com/ElementsProject/lightning/pull/5825
+    plugin.listpeerchannels = False
+    if len([c for c in rpchelp if c["command"].startswith("listpeerchannels ")]) == 0:
+        plugin.listpeerchannels = True
+
```

Comment:
we should have inside the init callback the value for `deprecated-api` value maybe it is better to look at this value instead to look at the help result? or you already skip this solution for some reason that I'm missing right now?

Reply:
normally I check for the `getinfo` version and parse that. But in this case I can't, since 23 is not yet released and the code that checks for version >= 23 would fail to do so (even though the codebase is already newer than 22)

Reply:
And the 'deprecated-api' value check will only hold for 6 month or so